prep-html-relative-pathing/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <div><a href="/contact-us/contact-us.html">Click here to go to the contact page</a></div>
```

Comment:
This is pathing from the root of your application. Please make use of a relative path, which starts with either `../` or the file or folder name you want to access. Let me know if you have any questions!